### PR TITLE
Add formatting changes for Github compatability

### PR DESCRIPTION
**Pull request by DavisC0801**
## Description

This pull request contains minor formatting changes for the Readme. 

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,8 @@ discard: [ordered array card ids]
 - - `get api/v1/game_state/1` - Returns the game state for game ID of 1.
 - Example Response:
 - - `Status: 200`
--  ~~~~
+-  
+~~~~
```
Comment (BrennanAyers) [inline README.md]: @DavisC0801 Can you change these characters to the ``` to follow the formatting convention we've been using in the rest of the documentation?

Comment (DavisC0801) [reply README.md]: Done, issue was with bullets and not the codeblock tag.

File: README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ discard: [ordered array card ids]
 - - `get api/v1/game_state/1` - Returns the game state for game ID of 1.
 - Example Response:
 - - `Status: 200`
--  ~~~~
+```
```
Comment (BrennanAyers) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -161,13 +161,21 @@ discard: [ordered array card ids]
         "id_list":[1041,1042,1043,1044,1045,1046,1047,1048]
       }
     ],
-    "playerOrder":["Player_1_Name"],
+    "playerOrder":["Player_1_Name","Player_2_Name"],
```
Comment (BrennanAyers) [inline README.md]: Good catch!

**Pull request merged**